huggingface_basics.py

```
@@ -0,0 +1,454 @@
+bert_output = bert_model(**encoded_input)
+# GPT-2 모델 활용
+gpt_model = GPT2LMHeadModel.from_pretrained('gpt2')
+gpt_tokenizer = AutoTokenizer.from_pretrained('gpt2')
+encoded_input = gpt_tokenizer(text, return_tensors='pt')
+gpt_output = gpt_model(**encoded_input)
+
+
+# In[ ]:
+
+
+bert_output #출력값: 입력 텍스트의 각 토큰과 전체 시퀀스를 나타내는 고차원의 벡터(은닉 상태)
```

Ruff (0.13.3)

Found useless expression. Either assign it to a variable or remove it.

(B018)